xarray/backends/rasterio_.py

```diff
@@ -176,7 +176,7 @@ def open_rasterio(filename, parse_coordinates=None, chunks=None, cache=None, loc
 
         from affine import Affine
         da = xr.open_rasterio('path_to_file.tif')
-        transform = Affine.from_gdal(*da.attrs['transform'])
+        transform = Affine(*da.transform)
```

Comment:
I think we should prefer `da.attrs["transform"]` because otherwise it is impossible to know whether `"transform"` is an attribute or a coordinate.

Reply:
Makes sense
```suggestion
        transform = Affine(*da.["transform"])
```